<launch>
  <param name="device_id" value="4"/>
  <param name="width" value="1920"/>
  <param name="height" value="1920"/>
  <node name="camera" pkg="camera-controller" type="camera.py" output="screen"/>
  <node name="fisheye_calibrate" pkg="camera-controller" type="fisheye_calibrate.py" output="screen" cwd="node"/>
  <node name="image_view" pkg="image_view" type="image_view" output="screen">
    <remap from="image" to="calib_image" />
  </node>
</launch>
